src/client/core/aad/authentication/authentication.service.spec.ts

```diff
@@ -32,42 +37,32 @@ describe("AuthenticationService", () => {
         let result: AuthorizeResult | null;
         let error: AuthorizeError | null;
         let promise;
-        beforeEach(() => {
+        beforeEach(async () => {
             result = null;
             error = null;
             const obs = userAuthorization.authorize("tenant-1");
             promise = obs.then((out) => result = out).catch((e) => error = e);
```

Comment:
Should this be await obs instead of obs.then?

Reply:
I had thought that too. But some of the unit tests need to set up certain expectations before the promise is resolved.